.github/workflows/shisen_ci.yml

```diff
@@ -0,0 +1,23 @@
+name: Shisen CI
+on:
+  pull_request:
+    branches:
+      - master
+  push:
+    branches:
+      - master
+      
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    name : Build the Project
+
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-node@v2
+      with:
+      node-version: '14'
```

Comment:
Indent the `node-version` too, as it lies inside the `with:` parameter, see [this guide](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idstepswith).